python/lsst/sconsUtils/vcs/git.py

```diff
@@ -23,11 +23,13 @@ def guessVersionName():
     if not os.path.exists(".git"):
         state.log.warn("Cannot guess version without .git directory; version will be set to 'unknown'.")
         return "unknown"
+    desc = utils.runExternal("git describe --tags --always", fatal=True)
```

Comment:
Now that I think about it further, I think that changing this to:

```suggestion
    desc = utils.runExternal("git describe --tags --always --dirty", fatal=True)
```

will fix this for us automatically.